src/components/CartButton/CartButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import shopping_cart from '../../img/shopping_cart.svg'
+import './CartButton.css'
+
+const cartImageStyle = {
+	width: "40px",
+	height: "40px",
+  backgroundImage: `url(${shopping_cart})`,
+  backgroundSize: 'cover',
+}
```

Comment:
since cartImageStyle is styling, you could move it into the `.cart-wrapper` css styles. That way we'll have all of our our styles in one spot